install/kubernetes/helm/subcharts/gateways/templates/role.yaml

```diff
@@ -0,0 +1,16 @@
+{{- range $key, $spec := .Values }}
+{{- if eq $key "istio-ingressgateway" }}
+{{- if $spec.ingressSds.enabled }}
```

Comment:
Same here - ingress is very confusing ( it's the k8s API we're deprecating ). 

Just 'sds.enabled' - this is already in context of a gateway config, no need to be redundant.